Samples/Windowing/cpp-winui/SampleApp/MainWindow.xaml.cpp

```diff
@@ -87,7 +87,7 @@ namespace winrt::SampleApp::implementation
 
         // Get the WindowId for our window
         winrt::WindowId windowId;
-        winrt::GetWindowIdFromWindowHandle(hWnd, &windowId);
```

Comment:
Curious, why this change?

Reply:
Honestly, I'm not sure if this is something that changed between versions of WindowsAppSDK, but after updating to WAS 1.0.0-preview, this sample would not build correctly, since it was saying "GetWindowIdFromWindowHandle() didn't take two parameters." I saw the definition of that API and it indeed only took 1 parameter, and returned a WindowID. 

I figured since it wasn't a huge change, I might do it so it would build and run correctly :)

Reply:
Got it! Yeah, good to do!

Since you're also modifying the Windowing samples, adding Samantha Song and Eve Mwangi would be good.